packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -85,6 +99,8 @@ function BasicColorList({
             pattern={pattern}
             isSelected={isSelected}
             isDisabled={isDisabled}
+            tabIndex={0 === i ? 0 : -1}
```

Comment:
If you open it for the background, the tabIndex should default to 0 on the color that's already highlighted for the background (`isSelected`) and not just the first item. However, this is actually doubly broken now, because it sets tabIndex=0 on the first item, which is transparent, which is disabled for the background, so you can't tab to the list of default colors at all.

Furthermore, I have a semitransparent color as my second saved color, so if tabbing to those, I can't use arrow keys, as the second item is not focusable. `useRovingTabIndex` should probably be updated to ignore/skip disabled buttons.

Reply:
Hm, I assumed it did skip disabled items, I think we had this problem before already somewhere 🤔